assembly/assembly-main/src/assembly/bin/che.sh

```diff
@@ -375,11 +375,45 @@ get_docker_ready () {
     if [ "${WIN}" == "true" ] || [ "${MAC}" == "true" ]; then
       launch_docker_vm
     else
-      # If Linux and docker ps fails, then this likely a file permissions issue.
-      error_exit "Running 'docker' succeeded, but 'docker ps' failed. `
-                 `This usually means that docker cannot reach its daemon. `
-                 `On Mac and Linux, check the read / write permissions on '/var/run/docker.sock'. `
-                 `Consider running 'sudo chmod 777 /var/run/docker.sock'."
+
+      # CHE-1202: Improve error messages in case of docker ps failure
+      # Verify that /var/run/docker.sock has owner and group read / write permissions
+      PERMS=$(stat -c %A /var/run/docker.sock)
```

Comment:
1. Even if docker.sock has owner/group rw permission it doesn't mean that current user is owner or included in needed group.
2. docker.sock can have rw permission for others.
3. `sudo chmod 660 /var/run/docker.sock` will help only when user has needed id or gid